Components/avatar/avatar.css

```diff
@@ -0,0 +1,80 @@
+.avatar-xl-box{
+    width: 6rem;
+    height: 6rem;
+    border-radius: 50%;
+    overflow: visible;
+    margin: 0.5rem;
+    position: relative;
+
+}
+
+.avatar-l-box{
+    width: 4rem;
+    height: 4rem;
+    border-radius: 50%;
+    overflow: visible;
+    margin: 0.5rem;
+    position: relative;
+    font-size: 12px;
+
+}
+
+.avatar-s-box{
+    width: 2rem;
+    height: 2rem;
+    border-radius: 50%;
+    overflow: visible;
+    margin: 0.5rem;
+    position: relative;
+    font-size: 8px;
+
+}
+
+.avatar-xs-box{
+    width: 1rem;
+    height: 1rem;
+    border-radius: 50%;
+    overflow: visible;

```

Comment:
I guess overflow by default value is visible. Any reason why you are defining it? Le me know your thoughts about this.

Reply:
There was an added scrollbar or some visual error it was fixed with overflow: visible. It looks completely normal in the website so it should be fine.